File: Java/#1313_decompressRLElist.java
class Solution {
    public int[] decompressRLElist(int[] nums) {
       int arraySize = 0;
        for(int i=0;i<nums.length;i+=2){
            arraySize+=nums[i];
        }
        int[] list = new int[arraySize];
        
        int start=0;
        for(int i=0;i<nums.length;i+=2){
            Arrays.fill(list,start,start+nums[i],nums[i+1]);
            start+=nums[i];
        }
        return list;
        
    }
}

File: Python/#1313_decompressRLElist.py
class Solution:
    def decompressRLElist(self, nums: List[int]) -> List[int]:
        res = []
        for i in range(0,len(nums),2):
            fre = nums[i]
            val = nums[i+1]
            for j in range(fre):
                res.append(val)
        return res
